model/model_training/custom_datasets/qa_datasets.py

```diff
@@ -343,16 +343,16 @@ def __init__(self, cache_dir) -> None:
                 # DO NOT change this
                 # its the data that had syntax error
                 explanation = data["explaination"]
-                self.pairs.append((joke, explanation))
+                self.pairs.append(DatasetEntry([joke], [explanation]))
 
         if len(question) > 0 and len(answer) > 0:
-            self.pairs.append((question, answer))
+            self.pairs.append(DatasetEntry([question], [answer]))
```

Comment:
You mentioned already that question and answer are not use, so please remove them and also this `if` statement.

Reply:
Thank you for the review.
Will remove it.

Reply:
applied the suggested changes please check.